lib/image-stream/handlers.js

```diff
@@ -161,6 +163,17 @@ module.exports = {
         },
         transform: new PassThroughStream()
       };
+    }).catch((error) => {
+      if (/invalid footer/i.test(error.message)) {
+        throw errors.createUserError({
+          title: 'Invalid image',
+          description: `There was an error reading "${path.basename(file)}". `
+            + 'The image does not appear to be a valid Apple Disk Image, or may have the wrong filename extension.\n\n'
```

Comment:
> Are we sure that any error messages containing /invalid footer/i will always be related to Apple Disk Images?

In this case, yes, because this catch is in the dmg handler ;)

> Would it be worth tweaking the message slightly to ... a valid Apple Disk Image (DMG), ...?

Yeah, that's probably better